### PR TITLE
Fix typo in title and better narrative

### PR DESCRIPTION
**Pull request by resouer**
A better narrative for https://github.com/helm/charts/pull/15004/, and fix typo in title

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
Just minor doc fix
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by resouer** [comment]
Kindly pint @mattfarin

File: README.md [hunk 1]
```diff
@@ -100,6 +100,6 @@ To provide that support the API versions of objects should be those that work fo
 
 This project is still under active development, so you might run into [issues](https://github.com/helm/charts/issues). If you do, please don't be shy about letting us know, or better yet, contribute a fix or feature.
 
-## Happy helming in China
+## Happy Helming in China
 
-If you are in China, there are some problems to use Helm Charts directly (e.g. gcr.io, googleapis.com etc), you can use this mirror repo at https://github.com/cloudnativeapp/charts.
+If you are in China, there are some problems to use upstream Helm Charts directly (e.g. images hosted on `gcr.io`, `quary.io`, and Charts hosted on `googleapis.com` etc), you can use this mirror repo at https://github.com/cloudnativeapp/charts which automatically sync & replace unavailable image & repo URLs in every Chart.
```
Comment (cpanato) [inline README.md]: should be `quay.io` not `quary.io`

Comment (resouer) [reply README.md]: Ooops, fixed

**Comment by resouer** [comment]
kindly ping @cpanato, the typo has been fixed, PTAL.

**Review by cpanato — APPROVED** [review]
/approve
/lgtm

**Pull request merged**